docs/sidebars.js

```diff
@@ -24,5 +24,7 @@ module.exports = {
       "dev-tests",
     ],
     "Dev Guides": ["dev-guide-new-shield", "dev-guide-usb-logging"],
+
+    Troubleshooting:["troubleshooting"],
```

Comment:
I think I'd rather this at the end of the `"Getting Started" section instead. Thoughts?

I also think there's a conflict here, you'll need to rebase/merge up w/ `main`.